scripts/dockerfiles/Dockerfile.build

```diff
@@ -11,10 +11,13 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-FROM golang:1.7
+FROM golang:1.9
 MAINTAINER Amazon Web Services, Inc.
 
-RUN mkdir /out
+RUN apt-get update &&\
+    apt-get install -y gcc-mingw-w64 &&\
+    mkdir /out
```

Comment:
I'd rather this `mkdir` be in a separate `RUN` directive since it isn't related to package installation.

Reply:
I'm thinking about simply putting it in the makefile itself, since several different targets expect this folder to exist.